app/README.md

```diff
@@ -1,80 +1,118 @@
 
-For typechecking, this application is leveraging Next.js' [incremental typechecking](https://nextjs.org/docs/basic-features/typescript#incremental-type-checking). NextJS will run type checking as a part of `next build`-- Although it is still recommended that we set up type checking using our code editor for development. In VSCode, do so by adding the following to your `.vscode/settings.json` file
+## 🖼️ Storybook
 
-```
-"typescript.validate.enable": true
-```
+Storybook is a [frontend workshop](https://bradfrost.com/blog/post/a-frontend-workshop-environment/) for developing and documenting pages and components in isolation. It allows you to render the same React components and files in the `src/` directory in a browser, without the need for a server or database. This allows you to develop and manually test components without having to run the entire Next.js application.
+
+From the `app/` directory:
+
+1. `npm run storybook`
+2. Navigate to [localhost:6006](http://localhost:6006) to view
 
-Note: make sure TypeScript and Javascript Language Features are enabled in VS Code Extensions.
+Alternatively, you can run Storybook in a Docker container:
 
-### Tsconfig additions to auto-generated file
+1. From the root directory run `docker-compose exec nextjs npm run storybook`
 
-- `target: es6` -- nextJS set this valule to `es5` by default.
-- `allowJS: true` -- allows ts checks on javascript files.
-- `checkJS: true` -- works in tandem with `allowJS`. Alternative to adding `// @ts-check` at top of Js files-- instructs ts to check all js files.
-- `jsx: react-jsx` -- [Documentation on this option](https://www.typescriptlang.org/docs/handbook/jsx.html). This value updates the JSX mode that TS whips with. Updating this from "preserve" helped get jest tests into a passing state. TODO: figure out why.
-- `incremental: true` -- enables incremental typechecking. Incremental typechecking creates a series of `.tsbuildinfo` files to dave information from last compilation. This expedites type checking during build.
-- `baseUrl: "."` & `paths: { @pages/*: [pages/*] }` -- These two, in tandem, setup module path aliases for cleaner imports. To utilize this, import files like: `import Home from "@pages/index";`
+Other scripts:
 
-## Package.json
+- `npm run storybook-build` - Exports a static site to `storybook-static/`
 
-### Scripts
+## 🐛 Testing
 
-These can be run with `npm run [name of script]`. For more about npm scripts, see [here](https://docs.npmjs.com/cli/v8/using-npm/scripts).
+[Jest](https://jestjs.io/docs/getting-started) is used as the test runner and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) provides React testing utilities.
+
+Tests are manged as `.test.ts` (or `.tsx`) files in the the `tests/` directory.
+
+To run tests:
+
+- `npm test` - Runs all tests and outputs test coverage report
+- `npm run test-update` - Updates test snapshots
+- `npm run test-watch` - Runs tests in watch mode
```

Comment:
You can also run  e.g. `npm run test Layout` to run just one test so I'm wondering what the difference is. 
 
 watch mode would mean it would only run after edits to the page file? or the test file? or both?
 
 I'm wondering what would be most helpful to list in the readme - I've never personally used test-watch 😳

Reply:
ah I can clarify this. Watch mode will automatically run the tests any time you save changes. It also includes an interactive command line, so you can change which test files you're filtering by or update snapshots.

Reply:
Could we add a [link](https://jestjs.io/docs/cli#--watch) to the jest doc here to help explain what watch mode is? Or is it not worth the chance that the link gets out of date?

Reply:
Updated in https://github.com/navapbc/template-application-nextjs/pull/67/commits/60fbbdc8847be33d7657ad47a2b1ba5a7b206fce